### ISSUE TITLE
[Bug]: Test image pipeline“error log：ModuleNotFoundError: No module named 'timm”：Unable to generate vectors

### ISSUE THREAD

**Issue by jingkl** [towhee-io/towhee#345]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Only towhee/image-embedding-resnet50 and towhee/image-embedding-resnet101 can generate vectors, the rest cannot generate vectors：
pipeline：{    "towhee/image-embedding-efficientnetb5",
                       "towhee/image-embedding-efficientnetb7", 
                       "towhee/image-embedding-resnet101",
                       "towhee/image-embedding-resnet50", 
                       "towhee/image-embedding-swinbase",
                       "towhee/image-embedding-swinlarge", 
                       "towhee/image-embedding-vitlarge" }

error log：
1、<img width="1431" alt="截屏2021-12-25 下午2 24 08" src="https://user-images.githubusercontent.com/34296482/147378884-191d89f4-75e9-4490-b070-0f4d41c0e0bb.png">
2、<img width="1419" alt="截屏2021-12-25 下午2 23 37" src="https://user-images.githubusercontent.com/34296482/147378889-3cf2a5c5-3f90-40a8-a254-735fdb1776bd.png">
3、<img width="1407" alt="截屏2021-12-25 下午2 25 28" src="https://user-images.githubusercontent.com/34296482/147378939-358ef362-1535-4e47-bf8c-9d3e1cb769bc.png">
4、<img width="1429" alt="截屏2021-12-25 下午2 24 19" src="https://user-images.githubusercontent.com/34296482/147378959-c538ba5e-9635-46cb-b8ba-6bc72997458e.png">
5、
<img width="1375" alt="截屏2021-12-25 下午2 22 54" src="https://user-images.githubusercontent.com/34296482/147378982-85ac11c6-428e-46e5-b642-8b52bab56b9a.png">

### Environment

```markdown
- Towhee version(e.g. v0.1.3 or 8b23a93):0.3.1.dev18
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

**Comment by jaelgu** [comment]
@jingkl I've added package dependencies in requirements.txt. Please test again now.

**Comment by jingkl** [comment]
My environment does not have automatically generated local folders @jaelgu

**Comment by jingkl** [comment]
I started a new environment to install towhee error
<img width="1435" alt="截屏2021-12-25 下午3 30 51" src="https://user-images.githubusercontent.com/34296482/147380013-202e936e-14b9-4398-953b-d02b45d6a764.png">

**Comment by jingkl** [comment]
blocked by ：#346

**Comment by jingkl** [comment]
Results of the new test：
<img width="1387" alt="截屏2021-12-25 下午4 28 59" src="https://user-images.githubusercontent.com/34296482/147380984-83a394f9-4174-4d95-b32f-e6d4e4d87c83.png">

**Comment by jingkl** [comment]
Finally I installed the appropriate package, but still reported an error
<img width="1431" alt="截屏2021-12-25 下午4 56 23" src="https://user-images.githubusercontent.com/34296482/147381527-763757ae-4a78-4a1a-bdf2-cfe768991fc9.png">

**Comment by jingkl** [comment]
<img width="1431" alt="截屏2021-12-25 下午6 23 05" src="https://user-images.githubusercontent.com/34296482/147382983-3bae222b-52b1-446a-9566-9dc25ad1b864.png">

**Comment by jingkl** [comment]
I think we should add a message indicating the success of the vector generation

**Comment by fzliu** [comment]
@jingkl Please open a new issue/enhancement for that.

**Comment by jingkl** [comment]
> @jingkl Please open a new issue/enhancement for that.

ok
